manim/mobject/svg/text_mobject.py

```diff
@@ -96,22 +100,27 @@ def __init__(self, text, **config):
     def get_space_width(self):
         size = self.size * 10
 
-        dir_name = file_writer_config['text_dir']
-        file_name = os.path.join(dir_name, "space") + '.svg'
+        dir_name = config["text_dir"]
```

Comment:
```suggestion
        dir_name = file_writer_config["text_dir"]
```

This needs to be changed to use file_writer_config. It showed up as a merge conflict, and I tried to resolve it, but it ended up reversing all the formatting changes black made and I had to revert what I did. Sorry 'bout that.